index.bs

```diff
@@ -4542,6 +4655,53 @@ Issue: We might want to have a more sophisticated filter system than just a
 
 </div>
 
+#### The script.removeLoadScript Command ####  {#command-script-removeLoadScript}
+
+The <dfn export for=commands>script.removeLoadScript</dfn> command removes a
+[=load script=].
+
+<dl>
+   <dt>Command Type</dt>
+   <dd>
+    <pre class="cddl remote-cddl">
+      ScriptRemoveLoadScriptCommand = {
+        method: "script.removeLoadScript",
+        params: ScriptRemoveLoadScriptParameters
+      }
+
+      ScriptRemoveLoadScriptParameters = {
+        script: LoadScript
+      }
+    </pre>
+   </dd>
+   <dt>Return Type</dt>
+   <dd>
+    <pre class="cddl">
+      EmptyResult
+    </pre>
+   </dd>
+</dl>
+
+<div algorithm="remote end steps for script.removeLoadScript">
+The [=remote end steps=] given |session| and |command parameters| are:
+
+1. Let |script| be the value of the "<code>script</code>" field in |command
+   parameters|.
+
+1. Let |load script map| be |session|'s [=load script map=].
+
+1. If |load script map| does not <a for=map>contain</a> |script|, return
+   [=error=] with [=error code=] [=invalid argument=].
+
+   TODO: Should this be a "no such script" error?
```

Comment:
We would raise invalid argument for not supported types, so maybe having a dedicated error like for invalid browsing context would be good to have.

How do we want to return the failure given that the CDDL expects an `EmptyResult` (this actually also applies to other commands)?

Reply:
Protocol-level errors like this have a totally different return type compared to commands that are run successfully; that's already handled in the transport section. Script errors are different because that error is itself a response to the command, not something that prevented the command running.